sovrin_client/agent/agent_prover.py

```diff
@@ -64,13 +64,16 @@ def sendReqClaim(self, link: Link, schemaKey):
 
         # TODO link.invitationNonce should not be used here.
         # It has served its purpose by this point. Claim Requests do not need a nonce.
+        public_key = await self.prover.wallet.getPublicKey(ID(schema_key))
+        schema = await self.prover.wallet.getSchema(ID(schema_key))
+
         op = {
```

Comment:
Do we have a test checking that CLAIM_REQ from libsovrin can be processed by this code?